ESLint-Formatter.py

```diff
@@ -143,11 +143,37 @@ def get_node_path():
     return node
 
   @staticmethod
+  def get_local_eslint(root, folders):
+    pkg = os.path.join(root, 'package.json')
+    esl = 'node_modules/.bin/eslint'
+
+    # find package.json first
+    if os.path.isfile(pkg):
+      # get eslint from local node_modules
+      return os.path.join(root, esl)
```

Comment:
findup is a pretty common pattern / function, looking at how it is typically implemented might help make sure we don't miss an edge case here.